pino.js

```diff
@@ -258,7 +259,7 @@ Pino.prototype.child = function child (bindings) {
   return new Pino(
     bindings.level || this.level,
     this.stream,
-    this.serializers,
+    extend(this.serializers, bindings.serializers),
```

Comment:
what effect does this have on perf, is there a less obvious but more performant way to achieve this?